doc/source/dev/development_environment.rst

```diff
@@ -35,10 +34,14 @@ any) found on current PYTHONPATH.
 When specifying a target using ``-s``, ``-t``, or ``--python``, additional
 arguments may be forwarded to the target embedded by ``runtests.py`` by passing
 the extra arguments after a bare ``--``. For example, to run a test method with
-the ``--pdb`` flag forwarded to nose, run the following::
+the ``--pdb`` flag forwarded to pytest, run the following::
 
     $ python runtests.py -t numpy/tests/test_scripts.py:test_f2py -- --pdb
 
+To `match test names using regexp`_, pass the ``-k`` argument to pytest::
+
+    $ python runtests.py -v -t numpy/core/tests/test_nditer.py -- -k test_iter_c_order
```

Comment:
Would be a better example with `test_iter_.*` or some actual regex

Reply:
fixed